tests_my/testcases/dashboard/login_test.py

```diff
@@ -0,0 +1,40 @@
+from selenpy.support import browser
+from tests_my.pages.dashboard.login_page import LoginPage
+from tests_my.testcases.test_base import TestBase
+from tests_my.testcases.utils import constants, common
+import pytest
+from tests_my.pages.dashboard.dashboard_page import DashboardPage
+import logging
+
+class Test_Login(TestBase):
+
+    login_page = LoginPage()
+    dashboard_page = DashboardPage()
+
+    @pytest.fixture()
+    def setup_class(self):
```

Comment:
Setup method will run before test cases. But your method is clean_up

Reply:
I updated